artifactory/utils/utils_test.go

```diff
@@ -61,3 +61,17 @@ func testGetRtMajorVersion(t *testing.T, version string, expected int) {
 	assert.NoError(t, err)
 	assert.Equal(t, expected, major)
 }
+
+func TestCreateMetadataServiceManager(t *testing.T) {
+	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.RequestURI == "api/v1/query" {
+			content := []byte(`{"query":"queryBody"}`)
+			_, err := w.Write(content)
+			assert.NoError(t, err)
+		}
+	}))
+	defer testServer.Close()
+	serviceManager, err := CreateMetadataServiceManager(&config.ServerDetails{MetadataUrl: testServer.URL + "/"}, false)
+	assert.NoError(t, err)
+	assert.NotEmpty(t, serviceManager)
+}
```

Comment:
this test doesn't check anything, it just creates a server without calling it. you can just avoid it as this is being tested by the client